app/admin/allocation_chart.rb

```diff
@@ -11,9 +13,33 @@
           column(I18n.t('name'), :user)
           column(I18n.t('skills')) { |allocation| allocation.user.skills.map(&:title).join(', ') }
           column(I18n.t('client'), :project)
-          column(I18n.t('allocated_until'), :end_at)
+          column(I18n.t('allocated_until'), class: 'allocated-column') do |allocation|
+            build_allocation_status_cell(allocation)
+          end
         end
       end
     end
   end
 end
+
+def build_allocation_status_cell(last_allocation)
+  allocation_status = ActiveAdmin::AllocationChartHelper.allocation_status_for(last_allocation: last_allocation)
+
+  unless last_allocation.end_at.nil?
```

Comment:
nothing wrong here! but I would suggest avoiding using the inverse logic of `unless`, since it tends to be a bit harder to read because we're more used to `if`s. So what do you think of changing this to something like:
```ruby
if last_allocation.end_at
  return column_cell_base(content: last_allocation.end_at, allocation_status: allocation_status)
end
```